js/index.js

```diff
@@ -40,3 +40,70 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let midImg = document.getElementById("cta-img");
+midImg.setAttribute('src', siteContent["cta"]["img-src"])
+
+let lastImg = document.getElementById("middle-img");
+lastImg.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
```

Comment:
getElementBy is the old school way to do it.  querySelector usually is the way to go. both work though of course.